src/shared/lib/styledComponents/theme.ts

```diff
@@ -0,0 +1,100 @@
+import { css } from "styled-components";
+export const theme = {
+  fonts: {
+    head3: css`
+      font-size: 28px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 32px;
```

Comment:
```suggestion
      line-height: 40px;
```

P1. 피그마 데이터 정의와 line-height 값이 다른 것 같은데 의도된 부분일까?

P3. 이름을 headline에서 head로 변경한 이유도 궁금하네. 단순히 변수명을 축약하기 위함이라면 원활한 소통을 위해 토큰 이름은 통일하는 게 좋을 것 같아서!

![image](https://github.com/user-attachments/assets/4ab7bab6-53dd-4961-8423-75fac542a957)

Reply:
맞아 사실 고민하다가 축약했었는데 완전 통일시키는게 팀차원 소통에서 좋겠구나! Headline으로 변경할게

Reply:
[feat: fonts명 피그마와 통일 및 오타 수정 (추후 디자인토큰 변경사항 적용 필요)](https://github.com/Nexters/DDD-web/pull/10/commits/aeca5ad469cd31e2a2e3ba46a812b3fdf4a62bfe)

Reply:
> 디자이너 왈: 이거 아직 정리한거 아니고 임의로 넣어둔거라서~ 수치 맞게 들어가있는지 체크 안했어 
정리하고 노티 줄게

추후 속성 값 변경 필요